src/Components/Cart/Cart.js

```diff
@@ -1 +1,360 @@
+
+      for (let i = 0; i < cartProducts.length; i++) {
+        prices.push(cartProducts[i].quantity * cartProducts[i].price);
+      }
+      setTotalPrice(prices.reduce((prev, curr) => prev + curr).toFixed(2));
+    }
+  }, [cartProducts]);
+
+  return (
+    <Container handleCartDisplay={handleCartDisplay}>
+      <div className="wrapper">
+        <TitleBox>
+          <div>
+            <h1>
+              Your Cart
+              <span>({totalQuantity})</span>
+            </h1>
+            <GrClose className="closeIcon" onClick={handleCartClose} />
+          </div>
+        </TitleBox>
+        <div className="wrapper">
+          <EmptyCart cartProducts={cartProducts}>
+            <span>your cart is empty</span>
+            <PaymentBtn>go back to shopping</PaymentBtn>
+          </EmptyCart>
+          <CartBox cartProducts={cartProducts}>
+            <ProductList>
+              {cartProducts.length > 0 &&
+                cartProducts.map((product) => {
+                  return (
+                    <CartItem
+                      product={product}
+                      // handleQuantity={(productId, option) =>
+                      //   handleQuantity(productId, option)
+                      // }
+                      handleCartListClick={(productId, option) =>
+                        handleCartListClick(productId, option)
+                      }
+                    />
+                  );
+                })}
+            </ProductList>
+            <RecommendBox>
+              <h2>You may also like</h2>
+              <div className="productContainer">
+                {cartRecommends.length > 0 &&
+                  cartRecommends.map((data) => {
+                    return <RecommendProduct data={data} />;
+                  })}
+              </div>
+            </RecommendBox>
+            <TotalBox>
+              <span>TOTAL</span>
+              <span className="price">
+                {totalPrice}
+                <span className="euro">€</span>
+              </span>
+            </TotalBox>
+            <PaymentBtn onClick={handlePayment}>continue to payment</PaymentBtn>
+            <ShippingDisplayBox totalPrice={totalPrice}>
+              <div className="conditionBox">
+                <div className="pricePercent"></div>
+              </div>
+              <h4>
+                {totalPrice > 39
+                  ? "Yes! Free shipping it is!"
+                  : "FREE SHIPPING WITH ALL(B)RIGHT AND FOR ORDERS OVER 39€"}
+              </h4>
+            </ShippingDisplayBox>
+          </CartBox>
+        </div>
+      </div>
+    </Container>
+  );
+};
+export default Cart;
+
+const Container = styled.section`
+  height: 100vh;
+  width: 50%;
+  padding: 2.78vw 4.17vw;
+  position: fixed;
+  top: 0;
+  right: -50%;
+  z-index: 150;
+  background-color: ${(props) => props.theme.yellow};
+  overflow-y: auto;
+  ${(props) =>
+    props.handleCartDisplay !== null &&
```

Comment:
```suggestion
    props.handleCartDisplay &&
```
null이 아니면 -> true이면 이라는 조건으로는 안될까요?